so.notion.Notion.yaml

```diff
@@ -0,0 +1,142 @@
+
+  - name: p7zip
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/p7zip/p7zip_16.02_src_all.tar.bz2
+        sha256: 5eb20ac0e2944f6cb9c2d51dd6c4518941c185347d4089ea89087ffdd6e2341f
+      - type: patch
+        path: p7zip-16.02-gcc10.patch
+    build-commands:
+      - cd CPP/7zip/CMAKE; sh ./generate.sh
+      - cp -f makefile.linux_amd64_asm makefile.machine
+      - make all2
+      - make install DEST_HOME=/app
+    cleanup:
+      - /share/doc
+
+  - name: imagemagick
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --with-modules
+    sources:
+      - type: git
+        url: https://github.com/ImageMagick/ImageMagick.git
+        tag: 7.0.10-46
+    cleanup:
+      - /share/doc
+
+  - name: notion
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://desktop-release.notion-static.com/Notion%20Setup%202.0.11.exe
+        sha256: 4528bb49ab677eb73b84da3a60840f6d1081ff0afd5c2d64ef6247058a38c19d
+        dest-filename: notion.exe
+      - type: file
+        path: package-lock.json
+      - type: file
+        path: package.json
+      - type: file
+        path: so.notion.Notion.desktop
+      - type: file
+        path: so.notion.Notion.appdata.xml
+      - type: file
+        path: notion-2.0.11-platform.patch
+      - generated-sources.json
+      - type: script
+        dest-filename: notion
+        commands:
+          - exec env TMPDIR=$XDG_CACHE_HOME zypak-wrapper /app/Notion-linux-x64/notion "$@"
+    build-options:
+      env:
+        npm_config_target: 11.0.4
+        npm_config_arch: x64
+        npm_config_target_arch: x64
+        npm_config_disturl: https://electronjs.org/headers
+        npm_config_runtime: electron
+        npm_config_build_from_source: 'true'
+        npm_config_nodedir: /run/build/notion/flatpak-node/node-gyp/electron-current
+        electron_config_cache: /run/build/notion/flatpak-node/electron-cache
+        ELECTRON_SKIP_BINARY_DOWNLOAD: 'true'
+    build-commands:
+      - 7z x notion.exe
+      - 7z x \$PLUGINSDIR/app-64.7z
+      - npx --offline --cache=flatpak-node/npm-cache asar@3.0.3 extract resources/app.asar app
+      - patch -t -p0 < notion-2.0.11-platform.patch
+      - rm -rf app/node_modules
+      - cp package.json package-lock.json app/
+      - npm install --prefix=app --offline --cache=flatpak-node/npm-cache
+      - |
+        npx --offline --cache=flatpak-node/npm-cache electron-packager@15.1.0 app Notion \
+          --platform linux \
+          --arch x64 \
+          --out $FLATPAK_DEST \
+          --electron-version 11.0.4 \
+          --executable-name notion \
+          --protocol notion \
+          --electron-zip-dir=flatpak-node/electron-cache
+      - install notion ${FLATPAK_DEST}/bin/
+      - convert app/icon.ico icon.png
+      - install -D icon-0.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
```

Comment:
256x256 is enough, flatpak will downscale it. SVG would be even better.

Reply:
I found an official SVG on their branding guidelines page and will include it in the next iteration.